drv/sidecar-front-io/build.rs

```diff
@@ -32,18 +19,13 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     )?;
 
     // Calculate a bitstream checksum and add it to the generated Rust file
```

Comment:
Nit: this comment is slightly wrong now, since we're echoing what the build system has told us what the checksum is